app.rb

```diff
@@ -31,8 +31,8 @@
   @payload = JSON.parse(payload_body)
 
   # If $required_plus_ones has been specified, change the default
-  if params['required_plus_ones']
-    NEEDED_PLUS_ONES = params['required_plus_ones']
+  if params['required_plus_ones'].to_i > 0
```

Comment:
Casting to an integer here will cause `nil` to have a `0` value no matter what.

Could do something like:

```ruby
if params['required_plus_ones'] && params['required_plus_ones'].to_i > 0
```